cmd/kusk/cmd/deploy.go

```diff
@@ -140,7 +144,7 @@ var deployCmd = &cobra.Command{
 					}
 
 					if len(api.Namespace) == 0 {
-						api.Namespace = envoyFleetNamespace
+						api.Namespace = "default"
```

Comment:
Just a small comment here might be useful to someone reading the code:

> kusk deploy deploys API to the kusk-system namespace, that should not be the case, we should deploy the api to default namespace

I was wondering why it was hard-coded to `"default"`.